clients/java/src/main/java/io/camunda/zeebe/client/api/command/ModifyProcessInstanceCommandStep1.java

```diff
@@ -21,9 +21,26 @@
 
 public interface ModifyProcessInstanceCommandStep1 {
 
-  ModifyProcessInstanceCommandStep2 activateElement(final String elementId);
+  /**
+   * Create an activate instruction for the given element id. The element will be created within an
+   * existing element instance of the flow scope.
```

Comment:
🔧 We might consider describing when a user should use the other activateElement method over this one.

Reply:
I'm still confused on when to use ancestor myself. Do you have an example on what we could write here?

Reply:
In case multiple instances of the ancestor element exist. In that case the user must select the ancestor. In C7 this was called [ancestor selection](https://docs.camunda.org/manual/7.17/user-guide/process-engine/process-instance-modification/#ancestor-selection-for-instantiation).